Lib/test/support/os_helper.py

```diff
@@ -756,8 +757,10 @@ def __len__(self):
     def set(self, envvar, value):
         self[envvar] = value
 
-    def unset(self, envvar):
-        del self[envvar]
+    def unset(self, envvar, /, *envvars):
+        """Unset one or more known environment variables."""
```

Comment:
Perhaps:
```suggestion
        """Unset one or more environment variables."""
```

Reply:
Ah yes, now it's not just "known" as I previously thought.